extensions/PhabBugz/Extension.pm

```diff
@@ -12,8 +12,26 @@ use strict;
 use warnings;
 use parent qw(Bugzilla::Extension);
 
+use Bugzilla::Constants;
+use Bugzilla::Extension::PhabBugz::Feed;
+use Bugzilla::Extension::PhabBugz::Logger;
+
 our $VERSION = '0.01';
 
+BEGIN {
```

Comment:
I'd rather see you just call Bugzilla::Extensions::PhabBugz->get_instance() (remove the _). That way the person reading the code will know which file to look in.